curriculum/challenges/english/25-front-end-development/quiz-bash-scripting/66f1afbd9998e9c985d8e73b.md

```diff
@@ -17,439 +17,454 @@ Answer all of the questions below correctly to pass the quiz.
+
+```bash
+print_arguments() {
+  echo "Argument 1: $1"
+  echo "Argument 2: $2"
+}
+
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+print_arguments("a", "b")
 
 ---
 
-Placeholder distractor 2
+print_arguments(a, b)
 
 ---
 
-Placeholder distractor 3
+print_arguments{a, b}
 
 #### --answer--
 
-Placeholder answer
+print_arguments "a" "b"
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a correct way to initialize array in bash?
 
 #### --distractors--
 
-Placeholder distractor 1
+`ARR=["item1" "item2" "item3"]`
 
 ---
 
-Placeholder distractor 2
+`ARR=[["item1" "item2" "item3"]]`
 
 ---
 
-Placeholder distractor 3
+`ARR={"item1" "item2" "item3"}`
 
 #### --answer--
 
-Placeholder answer
+`ARR=("item1" "item2" "item3")`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to print the third value of array in bash?
 
 #### --distractors--
 
-Placeholder distractor 1
+`echo ${ARR[3]}`
 
 ---
 
-Placeholder distractor 2
+`echo $ARR[2]`
 
 ---
 
-Placeholder distractor 3
+`echo $ARR[3]`
 
 #### --answer--
 
-Placeholder answer
+`echo ${ARR[2]}`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to take user input to variable in bash?
 
 #### --distractors--
 
-Placeholder distractor 1
+`input MY_VAR`
 
 ---
 
-Placeholder distractor 2
+`cin MY_VAR`
 
 ---
 
-Placeholder distractor 3
+`get MY_VAR`
 
 #### --answer--
 
-Placeholder answer
+`read MY_VAR`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the access rights to file 1.txt after executing the command `chmod 765 1.txt`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`rwxrwxrwx`
 
 ---
 
-Placeholder distractor 2
+`rwxr-xrw-`
 
 ---
 
-Placeholder distractor 3
+`r-xrw-rwx`
 
 #### --answer--
 
-Placeholder answer
+`rwxrw-r-x`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What keyword does the statement `if ... then ...` end with?
 
 #### --distractors--
 
-Placeholder distractor 1
+end
 
 ---
 
-Placeholder distractor 2
+done
 
 ---
 
-Placeholder distractor 3
+stop
 
 #### --answer--
 
-Placeholder answer
+fi
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What keyword does the statement `for ... do ...` end with?
 
 #### --distractors--
 
-Placeholder distractor 1
+end
 
 ---
 
-Placeholder distractor 2
+od
 
 ---
 
-Placeholder distractor 3
+stop
 
 #### --answer--
 
-Placeholder answer
+done
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to create comment line in bash?
 
 #### --distractors--
 
-Placeholder distractor 1
+`// comment`
 
 ---
 
-Placeholder distractor 2
+`!comment`
 
 ---
 
-Placeholder distractor 3
+`/* comment`
 
 #### --answer--
 
-Placeholder answer
+`# comment`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a correct line could be inserted to code to create for loop?
+
+```bash
+# INSERT CODE HERE
+do
+    echo $i
+done
+```
```

Comment:
````suggestion

```bash

do
    echo $i
done
```
````

We can't use comments as instructional content.